paas/install_client.go

```diff
@@ -123,23 +122,100 @@ func (c *InstallClient) Install(cmd *cobra.Command, options *kubernetes.Installa
 			return err
 		}
 	}
-
 	if err := downloadFuseMLCLI(c.ui); err != nil {
 		return err
 	}
 
+	extensions, err := options.GetOpt("extensions", "")
+	if err != nil {
+		return err
+	}
+	if err := c.InstallExtensions(extensions.Value.([]string), options); err != nil {
+		return err
+	}
+
 	c.ui.Success().WithStringValue("System domain", domain.Value.(string)).Msg("FuseML installed.")
 
 	return nil
 }
 
+// InstallExtensions installs given ML extensions
+func (c *InstallClient) InstallExtensions(extensions []string, options *kubernetes.InstallationOptions) error {
+	if len(extensions) == 0 {
+		return nil
+	}
+
+	extensionRepo, err := options.GetOpt("extension_repository", "")
+	if err != nil {
+		return err
+	}
+
+	for _, name := range extensions {
+		c.ui.Note().Msg(fmt.Sprintf("Installing extension '%s'...", name))
+
+		extension := deployments.NewExtension(name, extensionRepo.Value.(string))
+
+		err := extension.LoadDescription()
+		if err != nil {
+			return errors.New(fmt.Sprintf("Failed to load description of extension %s: %s", name, err.Error()))
```

Comment:
I think description can be optional, unless there is something depending on it?

Reply:
The description is not mean the value of description tag, but the description file, `description.yaml` which is crucial ... let me rephrase the message